kernel/realsense/d4xx.c

```diff
@@ -431,7 +431,7 @@ struct ds5_dfu_dev {
 	enum dfu_state dfu_state_flag;
 	unsigned char *dfu_msg;
 	u16 msg_write_once;
-	unsigned char init_v4l_f;
+	// unsigned char init_v4l_f;
```

Comment:
Why do we want to keep this as a comment? and also the reference to it below?
Is this needed?

Reply:
We want to resolve that in different way, leave it as reminder.